applications/crossbar/src/crossbar_services.erl

```diff
@@ -106,18 +95,12 @@ accepting_charges(Context, JObj, Services) ->
 %%--------------------------------------------------------------------
 -spec extract_items(kz_json:object()) -> kz_json:objects().
 extract_items(JObj) ->
-    kz_json:foldl(fun extract_items_from_category/3
-                  ,[]
-                  ,JObj
-                 ).
+    kz_json:foldl(fun extract_items_from_category/3, [], JObj).
```

Comment:
this is a formatting change. changes that affect how we extract the data will be part of KAZOO-4826